### ISSUE TITLE
[CoE Starter Kit - QUESTION] New Dashboard PBI August

### ISSUE THREAD

**Issue by albertocastro365** [microsoft/coe-starter-kit#6488]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hi,
I have some questions about the new versión of the Dashboard PBI.

1. Apps
This value is the total of the Power Apps. I can see a Measure "Production App" but I supose that is not applied.
![image](https://github.com/microsoft/coe-starter-kit/assets/57954993/6a3f55b3-b541-4ff0-be95-83f9ffb713b1)

2. Cloud Flows
![image](https://github.com/microsoft/coe-starter-kit/assets/57954993/e7a521f8-b92e-4fc7-89d5-e23e9eb700ef)
Can you explain us the difference between there visualizations? They seem the same data.

3. Desktop flows
![image](https://github.com/microsoft/coe-starter-kit/assets/57954993/2b41dc77-c43c-4534-bdd1-708e8aa4c06b)
My dashboard shows 28 desktop flows but in my organization there are several teams using RPA and I can't see here their desktop flows. What's meaning there 28?

4. Power Pages
![image](https://github.com/microsoft/coe-starter-kit/assets/57954993/94617251-c122-46ab-8a34-25122fae4688)
Why all values are 10?

5. App usage
![image](https://github.com/microsoft/coe-starter-kit/assets/57954993/a0c9ea2b-f764-4536-8c32-b8b8543ff0c8)
Any idea that why I don't have data of these sessiones?

THANKS

### What solution are you experiencing the issue with?

Core

### What solution version are you using?

August 2023

### What app or flow are you having the issue with?

Dashboard PBI

### What method are you using to get inventory and telemetry?

Data Export

**Comment by manuelap-msft** [comment]
Hello!

Sorry it took me so long to pick this up.

1. We have a measure that calculates production apps (based on MS criteria, a production app is an app with more than 5 unique users per month or more than 50 sessions per month). It's calculated in the Production App measure that you can find in the Key Measures folder. So there's no filter to be applied, as the calculation is done in the measure. 
2. Thanks for catching this - one is meant to be the Count of Flow IDs and one is meant to be the Count of Flow Owner IDs. We forgot to apply the Count (Distinct) to the second visual so that it would only count unique flow owners. Fixed for October!
3. That should be the total number of Desktop Flows. You may be hitting this limitation due to which not all Desktop Flows can be inventoried https://learn.microsoft.com/en-us/power-platform/guidance/coe/limitations#large-desktop-flows
4. Looks like we had a missing relationship between Environment and Power Pages here in the Data Export dashboard. Fixed for October. 
5. We relate the App table with the App Usage table here - the most likely cause is that apps got deleted from the environment, but we have old usage for them in the Audit Log table (or the Usage from Data Export). We'll filter this visual to not show blank app IDs.

**Comment by manuelap-msft** [comment]
Power Pages post update:


![Image](https://github.com/microsoft/coe-starter-kit/assets/55705669/4ca6ad56-ffcd-4ffc-b7a6-15abfef3a533)

**Comment by manuelap-msft** [comment]
Cloud Flows with the fix:


![Image](https://github.com/microsoft/coe-starter-kit/assets/55705669/69d41894-202b-4cc6-855b-3e5766d2398e)

**Comment by manuelap-msft** [comment]
App usage with the new filter:


![Image](https://github.com/microsoft/coe-starter-kit/assets/55705669/44b2f226-819b-4d7d-a680-3a829e0ab334)

**Comment by albertocastro365** [comment]
Hi @manuelap-msft,
sorry, but I don't understand your answer anbout point 1.

Yes, we have a measure "Production App" but we shouldn't use it to this visualization?
It's show ALL apps but the label it's Production apps.
